module/trace/metrics.go

```diff
@@ -9,6 +9,6 @@ var (
 	spanDurationMetric = promauto.NewHistogramVec(prometheus.HistogramOpts{
 		Name:    "span_duration_s",
 		Help:    "The duration of the Jaeger span in seconds",
-		Buckets: []float64{.001, .005, .01, .05, .1, .25, .5, 1, 2.5, 5, 10, 30},
+		Buckets: []float64{.001, .01, .1, .5, 5, 10},
```

Comment:
This will make many of our span metrics (for example collection node tx ingest->proposal) lose almost all their precision. This metric was always a bit of a kludge since tracing wasn't set up, but I don't think we should change this until we have a replacement: either re-implementing span metrics with purpose-configured bucket sizes or setting up tracing.